js/slider.js

```diff
@@ -0,0 +1,44 @@
+var inputLeft = document.getElementById("input-left");
```

Comment:
Refactor this to be a class or something with input object as argument.

`var` should be replaced with `let` or `const` according to value mutability.

Reply:
```suggestion
const inputLeft = document.getElementById("input-left");
```
I'll make it const. Didnt get it "Refactor this to be a class or something with input object as argument."

Reply:
It is much better to create a class named slider and create new instance each time you need a slider on page.

```JavaScript
const slider = new Slider(document.querySelector('input.slider'));
```

Everything you do with your input will be encapsulated inside class instance.

Reply:
> 
> 
> It is much better to create a class named slider and create new instance each time you need a slider on page.
> 
> ```js
> const slider = new Slider(document.querySelector('input.slider'));
> ```
> 
> Everything you do with your input will be encapsulated inside class instance.

done